tests/Arch/ModelsTest.php

```diff
@@ -46,7 +46,8 @@
         $instance = $model::factory()->create();
 
         $dates = collect($instance->getAttributes())
-            ->filter(fn ($_, $key) => str_ends_with($key, '_at'));
+            ->filter(fn ($_, $key) => str_ends_with($key, '_at'))
+            ->reject(fn ($_, $key) => in_array($key, ['created_at', 'updated_at']));
```

Comment:
what about `deleted_at` if the model has `SoftDeletes` trait?

Reply:
No plans for `SoftDeletes`!